frontend/public/components/storage/attach-storage.tsx

```diff
@@ -154,12 +154,14 @@ class AttachStorageForm extends React.Component<
       if (!container.volumeMounts) {
         container.volumeMounts = [];
       }
-      container.volumeMounts.push({
-        name: volumeName,
-        mountPath,
-        subPath,
-        readOnly: mountAsReadOnly,
-      });
+      if (mountPath) { //have to have a mount path to create a volumeMount
```

Comment:
If mount path is required, can we return early with an error? In what cases would this not be set?